lib/helpers.js

```diff
@@ -44,6 +44,22 @@ exports.validateAppName = function (name) {
 };
 
 /**
+ * Check reserved keywords for name of properties and relations
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.checkReserveWords = function(name) {
+  var reserveWords = ['constructor'];
```

Comment:
Looks like a typo, I think this should be `reservedWords`. Since it's a constant, I am proposing to move it to top-level and use an upper-case name:

```
var RESERVED_PROPERTY_NAMES = ['constructor'];
```